test/cpu_fusion.cpp

```diff
@@ -4024,4 +4024,39 @@ TEST(cpu_fusion, validate_fuse_gru_inputs)
         EXPECT_TRUE(test::all_close(cpu_results.at(i), int_results.at(i), 1.0e-4f, 1.0e-4f));
     }
 }
+
+TEST(cpu_fusion, mlir_matmul_bias)
+{
```

Comment:
Could we also add a flag for this pass so that we can use it with `mlir-opt` and then add a list test for this pattern?